### PR TITLE
LPS-178022 Cleanup TypeFacet for optional removal

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:search

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:search

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/type/facet/portlet/TypeFacetPortletPreferencesImpl.java [hunk 1]
```diff
@@ -77,15 +78,13 @@ public List<KeyValuePair> getAvailableAssetTypes(
 
 		String[] assetTypes = getAssetTypesArray();
 
-		String[] allAssetTypes = getAllAssetTypes(companyId);
-
 		if (ArrayUtil.isEmpty(assetTypes)) {
-			assetTypes = allAssetTypes;
+			return Collections.emptyList();
```
Comment (julschong) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/type/facet/portlet/TypeFacetPortletPreferencesImpl.java:80]: Need to recheck this with regression here: https://github.com/liferay-core-infra/liferay-portal/pull/4319#issuecomment-1518221415
after reviewing the the code, I changed this part back to returning empty list.

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by tinatian** [comment]
@julschong As we talked, please resend.

Thanks.
Tina

**Pull request closed without merging**